modules/gdscript/gdscript.h

```diff
@@ -121,18 +121,25 @@ class GDScript : public Script {
 	struct UpdatableFuncPtr {
 		List<GDScriptFunction **> ptrs;
 		Mutex mutex;
-		bool initialized = false;
+		bool initialized : 1;
+		bool transferred : 1;
```

Comment:
Do we not need to initialize values here? Or does this syntax do it for us somehow?

Reply:
We should initialize, I think, the values here.

Reply:
They are in the constructor just below, don't know if bitfields can be directly initialized

Reply:
I tried, but that requires C++20.